<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DcmSorter</class>
 <widget class="QMainWindow" name="DcmSorter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>531</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>显示属性</string>
    </property>
    <widget class="QSlider" name="layerAdjuster">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>491</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="blendAdjuster">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>491</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>调整当前层</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>60</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>调整当前混合系数</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>211</width>
     <height>534</height>
    </size>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="loadDicom">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>读取DCM图片</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="imageStat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>181</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>图片属性</string>
     </property>
     <widget class="QComboBox" name="qualityEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>很差</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>一般</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>很好</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="phaseEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>动脉期</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>静脉期</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>平衡期</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>平扫期</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="tumorEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>无肿瘤</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>肿瘤少或小</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>肿瘤多或大</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>62</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>图片质量</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>属于哪一期</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>肿瘤质量</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="wwwlstat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>270</y>
       <width>181</width>
       <height>141</height>
      </rect>
     </property>
     <property name="title">
      <string>调整窗宽窗位</string>
     </property>
     <widget class="QSlider" name="WWAdjuster">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="WLAdjuster">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>160</width>
        <height>22</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>90</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>调整窗位</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>调整窗宽</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="saveResult">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>630</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>保存结果</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>181</width>
       <height>191</height>
      </rect>
     </property>
     <property name="title">
      <string>特殊属性与描述</string>
     </property>
     <widget class="QCheckBox" name="isLiverMaskError">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>肝脏分割有误</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="descriptionEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>161</width>
        <height>121</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.Lucida Grande UI'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;在此输入个性化描述&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
